controllers/userController.js

```diff
@@ -137,24 +134,46 @@ module.exports = {
   },
   addAddress: async (req, res) => {
     try {
-      const { address, city, state, pinCode, phoneNum, type } = req.body;
-      const data = req.tokenData;
-
-      await Address.create({
-        user_id: data.user_id,
+      const { address, city, state, pin_code, phone_no } = req.body;
+      const addAddress = await addresses.create({
         address,
         city,
         state,
-        pinCode,
-        phoneNum,
-        type,
+        pin_code,
+        phone_no,
+        userId: req.body.id,
```

Comment:
get id field the same as other fields, do not write req.body.id

Reply:
resolved